src/core/config/CustomModesManager.ts

```diff
@@ -73,12 +74,102 @@ export class CustomModesManager {
 		return exists ? roomodesPath : undefined
 	}
 
+	/**
+	 * Strip BOM (Byte Order Mark) from the beginning of a string
+	 */
+	private stripBOM(content: string): string {
```

Comment:
I think we have a stripbom method elsewhere too - could we move to a shared utility file?